### ISSUE TITLE
add support for alt text

### ISSUE THREAD

**Issue by pkra** [pkra/mathjax-img#2]
From https://github.com/mathjax/MathJax-third-party-extensions/issues/25

> For accessibility, there should be a way to specify alt text.

See also https://www.w3.org/Math/draft-spec/mathml.html#chapter3_presm.mglyph for the spec.

**Comment by pkra** [comment]
Even more useful now that https://github.com/zorkow/speech-rule-engine/issues/80 is done.

**Comment by pkra** [comment]
I'm leaning towards another bracketed argument for this, i.e., second or fourth getBrackets(). Any opinions welcome.

**Comment by pkra** [comment]
Filed feature request upstream.
